packages/@okta/vuepress-site/docs/guides/configure-signon-policy/main/index.md

```diff
@@ -83,66 +84,92 @@ The following are step-by-step instructions to configure a global session policy
+1. In the text box, enter the dynamic zone name and then select it when it appears in the list.
 
-4. Configure IF conditions to define the authentication context for the rule. Select **Not in any of the following zones** from the **AND User’s IP is** dropdown list.
+1. Configure THEN conditions to define the authentication experience. For this use case, leave the default of **Allowed** for **THEN Access is**.
 
-    > **Note:** You can click the **Go to Network Zones** link to access the gateway settings that enable your choice of access. A [network zone](https://help.okta.com/okta_help.htm?id=ext_Security_Network) is a security perimeter used to limit or restrict access to a network based on a single IP address, one or more IP address ranges, or a list of geolocations. You can also create network zones using the [Zones API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/NetworkZone/).
+1. For **Establish the user session with**, select **Any factor used to meet the Authentication Policy requirements**.
 
-5. In the text box, start typing the dynamic zone name and then select it when it appears in the list.
+1. Ensure that MFA is **Required**. Users who sign in to your app from a non-US IP address are prompted for an additional factor.
 
-6. Configure THEN conditions to define the authentication experience. For this use case, leave the default of **Allowed after successful authentication** for **THEN Access is**.
+1. Select how users are prompted for a secondary factor in a given session. For example, select **At every sign in** so that users are prompted at every time they sign in.
 
-7. In the **Re-authentication frequency** section, select **Every sign-in attempt** for both **AND Password re-authentication frequency is** and **AND Re-authentication frequency for all other factors is**.
+1. Configure the time settings for the rule.
 
-8. Click **Save**.
+1. Click **Save**.
 
 > **Note:** You can [use the API](/docs/reference/api/policy/#authentication-policy) to assign an app to an authentication policy. You need the application ID and the policy ID for this API request. Make a `PUT /api/v1/apps/{appId}/policies/{policyId}` request. No HTTP body is necessary for the PUT request. Then, to check that the assignment was successful, make a `GET /api/v1/apps/{appId}` request and the response should contain information on the policy associated with the app.
 
+## Prompt for a passwordless sign-in flow
+
+In this example, create a policy that allows a specific group, **Project planners**, for example, to use a passwordless sign-in flow. To set up a passwordless global session policy, ensure that you have set up a [passwordless sign-in experience](https://help.okta.com/okta_help.htm?type=oie&id=ext-passwordless). Create an authenticator enrollment and authentication policy for the specific group. Then, create a global session policy.
```

Comment:
The link should stay in site and point to /docs/guides/oie-embedded-common-org-setup/aspnet/main/#set-up-your-okta-org-for-a-password-factor-only-use-case